src/core/url_utils.ts

```diff
@@ -51,8 +51,7 @@ export function getStudyRawConfigUrl(
   return (
     'https://github.com/search?type=code' +
     '&q=repo%3Abrave%2Fbrave-variations' +
-    '+path%3A%2F%5Eseed%5C%2Fseed.json%7C%5Estudies%5C%2F*.json5%2F+' +
-    `"%5C"name%5C"%3A+%5C"${encodeURIComponent(study)}%5C""`
+    `+path%3Astudies%2F+%22name%3A+%27${encodeURIComponent(study)}%27%22`
```

Comment:
no need for `.json5` here?

Reply:
We don't need it here, because it's a code pattern to look in brave-variations. 
But we need to update the URL above.